src/NeoSharp.Core/Cryptography/ECPoint.cs

```diff
@@ -51,10 +51,22 @@ private ECPoint()
         /// <param name="point">Point</param>
         public ECPoint(byte[] point)
         {
-            IsInfinity = false;
+            if (point == null)
```

Comment:
again this change xD

Reply:
dammm sorry ... after merge the other PR I resolve the conflict on this...
sorry!